doi-requests-commons/src/main/java/no/unit/nva/doi/requests/api/model/responses/DoiRequestSummary.java

```diff
@@ -1,83 +1,77 @@
+    private final String mainTitle;
+    private final Instant modifiedDate;
+    private final DoiRequest doiRequest;
+    private final String publisherId;
 
-    /**
-     * Constructor for DoiRequestSummary.
-     *
-     * @param identifier        publication identifier
-     * @param owner             publication owner
-     * @param doiRequest        publication doiRequest
-     * @param entityDescription publication entityDescription
-     */
     @JsonCreator
-    public DoiRequestSummary(@JsonProperty(value = "identifier", access = Access.WRITE_ONLY) UUID identifier,
-                             @JsonProperty(value = "owner", access = Access.WRITE_ONLY) String owner,
-                             @JsonProperty(value = "doiRequest", access = Access.WRITE_ONLY) DoiRequest doiRequest,
-                             @JsonProperty(value = "entityDescription", access = Access.WRITE_ONLY)
-                                 EntityDescription entityDescription) {
-        this.doiRequestStatus = doiRequest.getStatus();
-        this.doiRequestDate = doiRequest.getDate();
-        this.publicationIdentifier = identifier;
-        this.publicationTitle = entityDescription.getMainTitle();
-        this.publicationOwner = owner;
+    public DoiRequestSummary(
+        @JsonProperty("id") UUID id,
+        @JsonProperty("owner") String owner,
+        @JsonProperty("doiRequest") DoiRequest doiRequest,
+        @JsonProperty("mailTitle") String mainTitle,
+        @JsonProperty("modifiedDate") Instant modifiedDate,
+        @JsonProperty("publisherId") String publisherId) {
+        this.id = id;
+        this.owner = owner;
+        this.doiRequest = doiRequest;
+        this.mainTitle = mainTitle;
+        this.modifiedDate = modifiedDate;
+        this.publisherId = publisherId;
     }
-    
+
     /**
      * Creates DoiRequest summary from a publication.
      *
      * @param publication the publication.
      * @return a DoiRequestSummary.
      */
     public static DoiRequestSummary fromPublication(Publication publication) {
-        return new DoiRequestSummary(
-            publication.getIdentifier(),
+
+        return new DoiRequestSummary(publication.getIdentifier(),
             publication.getOwner(),
             publication.getDoiRequest(),
-            publication.getEntityDescription()
+            extractTitle(publication),
+            publication.getModifiedDate(),
+            publication.getPublisherId()
         );
     }
 
-    public DoiRequestStatus getDoiRequestStatus() {
-        return doiRequestStatus;
-    }
-
-    public Instant getDoiRequestDate() {
-        return doiRequestDate;
-    }
-
-    public UUID getPublicationIdentifier() {
-        return publicationIdentifier;
-    }
-
-    public String getPublicationTitle() {
-        return publicationTitle;
-    }
-
-    public String getPublicationOwner() {
-        return publicationOwner;
+    private static String extractTitle(Publication publication) {
```

Comment:
Why don't we move this as a getter on publication? Very odd this ask object about information etc.  I feel we should be able to ask Publication directly about its title. Not having to extract it and do stuff.

Reply:
This class will cease to exist in the following PRs.